css/styles.less

```diff
@@ -1,108 +1,186 @@
 @font-headings: 'Raleway';

```

Comment:
I'm noticing your font-sizes are off, your margins are off, and your widths are off.
please double check the design.

thanks

Reply:
Also there should be a gradient